packages/style-engine/class-wp-style-engine.php

```diff
@@ -185,7 +185,16 @@ final class WP_Style_Engine {
 			),
 		),
 		'dimensions' => array(
-			'minHeight' => array(
+			'aspectRatio' => array(
+				'property_keys' => array(
+					'default' => 'aspect-ratio',
+				),
+				'path'          => array( 'dimensions', 'aspectRatio' ),
+				'classnames'    => array(
+					'has-aspect-ratio' => true,
```

Comment:
We might need to check the actual value of the aspect-ratio property before applying the class, because it's being applied even when aspect-ratio is `unset`:

<img width="653" alt="Screenshot 2024-01-24 at 10 47 03 am" src="https://github.com/WordPress/gutenberg/assets/8096000/ba2194ca-223b-4f18-86b4-dc666759142e">

Reply:
Oh, good catch! I'll look into it 👀

Reply:
There might be a better way to do it, but I've gone with skipping the classname output in `dimensions.php` if the value wasn't originally set in https://github.com/WordPress/gutenberg/pull/56897/commits/7e14457ce2ec28ed2cbd8756ca85c17051284d5c

I _think_ that might be preferable to handling it in the style engine as it'd probably be a bit of a larger change to get the style engine to accept a callback to determine whether or not to output the classname, and what we're doing with the aspect ratio support is a bit different to all the other block supports. If we find we have another use case for this kind of skipping logic, we could always look at updating the style engine in a follow-up?

The change appears to be working okay for me now:

<img width="732" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/33135146-adc0-4dbc-99b3-c94f3665914b">

Let me know if there's another way you think we should try it, though! 🙂